docs/Developers/Writing_Good_Commits.rst

```diff
@@ -83,8 +83,10 @@ Example:
    -  **Short Description**
 
       - Starts with a capital letter.
-      - Written in imperative present tense (i.e. ``Add something``, not
-        ``Adding something`` or ``Added something``).
+      - Written in `present tense imperative mood
+        <https://en.wikipedia.org/wiki/Imperative_mood>`_.
```

Comment:
There are little references in the article to the present tense. I feel like this could just be `imperative mood`